app/views/layouts/_navbar.html.haml

```diff
@@ -0,0 +1,13 @@
+%nav.navbar.navbar-expand-lg.navbar-dark.bg-dark
+  .container-fluid
+    %a.navbar-brand{href: root_url} TODO
+    %button.navbar-toggler{"aria-controls" => "navbarNav", "aria-expanded" => "false", "aria-label" => "Toggle navigation", "data-bs-target" => "#navbarNav", "data-bs-toggle" => "collapse", type: "button"}
+      %span.navbar-toggler-icon
+    #navbarNav.collapse.navbar-collapse
+      %ul.navbar-nav
+        %li.nav-item
+          %a.nav-link.active{"aria-current" => "page", href: root_url} Lists
```

Comment:
Is there a Rails-ier way we could setup these links?

Reply:
You could have it all done through the ActionView helpers with `link_to` and then pass a hash of options for setting the classes and other markup elements.